src/AzSK.ADO/Framework/Configurations/SVT/ControlSettings.json

```diff
@@ -291,29 +291,50 @@
         "Environment",
         "SolutionName"
     ],
-    "RestrictedBroaderGroupsForBuild" : {
-      "Contributors":[
+    "RestrictedBroaderGroupsForBuild": {
+
+      "Contributors": [
+        "Retain indefinitely",
+        "Manage build qualities",
```

Comment:
There are no security risks associated with the permissions "Retain indefinitely" and "Manage build qualities," so we've decided to remove them from the excessive list. Please remove these two permissions. @SaikumarRagala , please confirm.